features/maximum_coverage_drop.feature

```diff
@@ -266,3 +266,97 @@ Feature:
     And the output should contain "Line coverage has dropped by 15.22% since the last time (maximum allowed: 0.00%)."
     And the output should contain "Branch coverage has dropped by 50.00% since the last time (maximum allowed: 0.00%)."
     And the output should contain "SimpleCov failed with exit 3"
+
+  @primary_coverage
```

Comment:
`@primary_coverage` isn't needed - the scenarios that were tagged `@branch_coverage` are that way so that they can be ignored on for instance JRuby see https://github.com/simplecov-ruby/simplecov/blob/b366225e7dcbf47382caab2442b3ed27e05397a8/features/support/env.rb#L27-L29

so these should probably be tagged with that as well :)

Reply:
This might also be why the JRuby tests are failing: https://github.com/simplecov-ruby/simplecov/pull/959/checks?check_run_id=1617681063

Reply:
Ah that makes a ton of sense! I hadn't used cucumber before this repo, so am pretty unfamiliar with it!